chesster.js

```diff
@@ -161,6 +152,17 @@ function update_channels(bot){
     });
 
 }
+function refreshCurrentRoundSchedules() {
+    var _45_45 = league.getLeague("45+45", config);
+    _45_45.getCurrentRoundSchedule(function(err, pairings) {
```

Comment:
you make a good point - we may want to make refresh of particular entities possible from slack.

this way when theino or mrlegs are updating pairings, they can send `@chesster refresh pairings`